src/commands/subdomain.rs

```diff
@@ -105,15 +128,48 @@ pub fn set_subdomain(name: &str, user: &GlobalUser, target: &Target) -> Result<(
     }
     let subdomain = Subdomain::get(&target.account_id, user)?;
     if let Some(subdomain) = subdomain {
-        let msg = if subdomain == name {
-            format!("You have previously registered {}.workers.dev", subdomain)
+        if subdomain == name {
+            let msg = format!("You have previously registered {}.workers.dev", subdomain);
```

Comment:
nit: (i realize this was the pre-existing verbiage)
```suggestion
            let msg = format!("You have already registered {}.workers.dev", subdomain);
```